app.js

```diff
@@ -1,10 +1,28 @@
+// Get dependencies
```

Comment:
since you are using `const`, `let`, & arrow notation else where in the code, i'm pretty sure that you are using es6. please replace all `var` with either `const` or `let` depending on its use case.